modules/rgbd/samples/kinfu_demo.cpp

```diff
@@ -2,13 +2,14 @@
 // It is subject to the license terms in the LICENSE file found in the top-level directory
 // of this distribution and at http://opencv.org/license.html
 
-// This code is also subject to the license terms in the LICENSE_KinectFusion.md file found in this module's directory
+// This code is also subject to the license terms in the LICENSE_KinectFusion.md file found in this
+// module's directory
 
-#include <iostream>
 #include <fstream>
-#include <opencv2/imgproc.hpp>
+#include <iostream>
 #include <opencv2/calib3d.hpp>
 #include <opencv2/highgui.hpp>
+#include <opencv2/imgproc.hpp>
 #include <opencv2/rgbd/kinfu.hpp>
```

Comment:
avoid unnecessary changes

Reply:
Reverted all the style changes in this file.